corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -229,6 +229,8 @@ hqDefine('registration/js/new_user.ko', [
 
         var _getDataForSubmission = function () {
             var password = self.password();
+            var captchaInput = $(containerSelector).find("[name='captcha_0']")[0].value;
+            var captchaAnsText = $(containerSelector).find("[name='captcha_1']")[0].value;
```

Comment:
nit: how about captchaString and captchaResponse or captchaAnswer?

Reply:
changed :+1: